### PR TITLE
Tests passes

### PR DESCRIPTION
**Pull request by cbrzn**
Here we still using the old client version (which is not compatible with node v12.16.1) - I was thinking that we can use for now the v10.15.3 if we don't want to use the new one yet

My opinion is that we should move to the new one and then move to the 2.0 - The reason is because when we move to 2.0, we will need to do the job of making it easier to the developer to deploy the docker container, so why delay it? Let's do it now 📦 - Your call sir @dOrgJelli

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -24,6 +24,7 @@
     "noImplicitThis": true,
     "noImplicitAny": true,
     "strictNullChecks": true,
+    "esModuleInterop": true
```
Comment (cbrzn) [inline tsconfig.json:27]: This is key to make the test pass, without it it shows the following error: `TypeError: Cannot read property 'createElement' of undefined`

**Comment by dOrgJelli** [comment]
Hey @cbrzn, I think we should complete the other changes before updating to client 2.0. Check all of the issues in the milestone that are ordered above "Update To Client 2.0". All of these are things we should do before: https://github.com/dOrgTech/DAOcomponents/milestone/3

File: tsconfig.json (hunk 1 above)
Comment (dOrgJelli) [reply tsconfig.json:27]: Nice find!

File: jest.config.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+module.exports = {
+  transform: {
+    ".(ts|tsx)$": "ts-jest"
+  },
+  testPathIgnorePatterns: ["/node_modules/", "/dist/", "/test/declarations.d.ts"],
+  testRegex: "(/test/.*(spec).*|\\.(test|spec))\\.(ts|tsx|js)$",
+  moduleNameMapper: {
+    "\\.(css)$": "identity-obj-proxy"
```
Comment (dOrgJelli) [inline jest.config.js:8]: Is this needed since we don't have any css files?

**Comment by dOrgJelli** [comment]
Hey also I'm not able to get the tests to pass on my end. We should probably add a .travis.yml file to this project and have travis run these tests.

Also let's remove the snapshots.

File: jest.config.js (hunk 2 above)
Comment (cbrzn) [reply jest.config.js:8]: Yes, if we remove it we get the following error:

```
 FAIL  test/contextValidation.spec.tsx
  ● Test suite failed to run
/Users/Cesar/apps/dorg/daostack/DAOcomponents/node_modules/loaders.css/loaders.css:14
    @-webkit-keyframes scale {
    ^

    SyntaxError: Invalid or unexpected token
```

**Comment by cbrzn** [comment]
>Hey also I'm not able to get the tests to pass on my end. We should probably add a .travis.yml file to this project and have travis run these tests.

The problem was that the `react` version and `react-test-rendender` wasn't the same, thanks to this post I fixed it before but looks like the `yarn.lock` still had different versions: https://stackoverflow.com/questions/48081735/react-test-renderer-error-typeerror-cannot-read-property-offsetwidth-of-unde

**Comment by cbrzn** [comment]
@dOrgJelli updated now sir! Looks good to merge for me, let me know if something else is needed 🧑‍🏭

**Pull request merged**